tuf/api/metadata.py

```diff
@@ -1572,8 +1577,7 @@ def get_roles(self) -> Iterator[str]:
             yield f"{self.name_prefix}-{suffix}"
 
     def is_delegated_role(self, role_name: str) -> bool:
-        """Determine whether the given ``role_name`` is in one of
-        the delegated roles that ``SuccinctRoles`` represents.
+        """Determine whether the given ``role_name`` is in one of the delegated roles represented by ``SuccinctRoles``.
```

Comment:
This exceeds our usual line length of 80 quite a bit. Not sure if we have agreed on a limit for docstrings. Apparently we are not enforcing one. Can pydocstyle help with this? I don't have very strong feelings about the exact limit, but I think it makes sense to align(-ish) with the rest of the code.

Reply:
I see black running which should have 88 chars default, but it looks like it works on docstrings in general. What about enabling [flake8](https://flake8.pycqa.org/en/2.5.5/index.html) as well? There are few violations reported, but could be fixable.

Reply:
We configure 80 for black (see pyproject.toml). Also, black doesn't touch docstrings. I wonder why pylint doesn't complain. 

Regardless, I'm fine with adding another linter (flake8), but it's not a priority. I think the current combo of linters+formatters works quite well.

Reply:
Okay, so let's keep it without flake8 in that case.